### PR TITLE
Fix/silence warning from flake8-bugbear.

### PR DESCRIPTION
**Pull request by frosch123**
None of the fixes are critical. But the noise in "make flake" annoys me :)

### CODE REVIEW

**Review by glx22 — APPROVED** [review]
LGTM

File: nml/actions/action2layout.py [hunk 1]
```diff
@@ -67,7 +67,7 @@ def write(self, file):
         file.newline()
         if len(self.sprite_list) == 0:
             file.print_dwordx(0)  # sprite number 0 == no sprite
-            for i in range(0, 5):
+            for _ in range(0, 5):
```
Comment (FLHerne) [inline nml/actions/action2layout.py:70]: I'm not a fan of this loop. What do you think of `file.print_data(b"\0" * 5)`, or just `file.print_data(b"\0\0\0\0\0")`?

**Review by FLHerne — COMMENTED** [review]
See one comment.

I considered nitpicking about some of the AssertionErrors where semantically ValueError might make sense, but meh.

File: nml/actions/action2layout.py (hunk 1 above)
Comment (glx22) [reply nml/actions/action2layout.py:70]: It's a dword so each value is written \dx00000000 (in NFO) or 00 00 00 00 (in GRF)

Comment (FLHerne) [reply nml/actions/action2layout.py:70]: Good point. Ignore this idea then.

**Pull request merged**